lxd/network/driver_common.go

```diff
@@ -551,6 +551,8 @@ func (n *common) bgpValidationRules(config map[string]string) (map[string]func(v
 			rules[k] = validate.Optional(validate.IsInRange(1, 4294967294))
 		case "password":
 			rules[k] = validate.Optional(validate.IsAny)
+		case "holdtime":
+			rules[k] = validate.Optional(validate.IsInRange(9, 65535))
```

Comment:
Because holdtime is three times bigger than keepalive timer I simply restrict it to make minimum 3sec keepalive timer.
Do you think is better to have minimum value of 0(zero) like in bgp 
 specifications?

Reply:
Any preference @stgraber ?

Reply:
What do you mean about needing to add a keep alive option as well?

Reply:
In general, network devices allow to configure both BGP timers and I start to implement ability to configure both option.
But at the end of the day I realized that it complicates configuration little bit.
To make things simple I've added low range of 9 seconds (keepalive is three times lower).
In case we set low bound to zero, maybe (I just consult with you) we should also add keepalive option, but I now thing lets stick one lonely holdtime parameter.